package.json

```diff
@@ -24,8 +24,8 @@
     "pg-format": "^1.0.0",
     "pg-promise": "^3.0.0",
     "postgrator": "^2.8.2",
-    "pouchdb-core": "^7.0.0",
-    "pouchdb-adapter-http": "^7.0.0",
+    "pouchdb-core": "7.0.0",
+    "pouchdb-adapter-http": "7.0.0",
```

Comment:
Revert this if you can - using `^` is best practice.

Without that you might end up with something like `medic-couch2pg` using a different version of pouchdb than `couch2pg` which may have unpredictable results.

Reply:
I made this change because both depend on `pouchdb-fetch` which depends on `fetch-cookie` which has an update that breaks the build on node 6 because it uses async/await. I can make the same change on  `medic-couch2pg` for consistency but can't think of another way to ensure it still works on node 6.

Reply:
Ok. It's fine to leave it as it is for now. Thanks for the explanation!

Reply:
I've reverted the change since we're no longer running tests over Node 6. Will revert the change on the `medic-couch2pg` PR as well.